### PR TITLE
[Loopring 3.6] update ERC1271 related code and add small improvements

### PR DESCRIPTION
**Pull request by dong77**
I have the following changes:

- sync SignatureUtil.sol with the implementation from Hebao1.1
- rename TransactionTypes: NEW_ACCOUNT => ACCOUNT_NEW, OWNER_CHANGE => ACCOUNT_TRANSFER, UPDATE_ACCOUNT => ACCOUNT_UPDATE
- remove BlockType enum.
- auto JS reformat
- change Operator contract to use OwnerManagable instable Claimable

### CODE REVIEW

File: packages/loopring_v3/contracts/impl/StatelessWallet.sol [hunk 1]
```diff
@@ -29,7 +29,7 @@ contract StatelessWallet
     using SignatureUtil        for bytes32;
 
     // bytes4(keccak256("transferOwnership(bytes,bytes)")
-    bytes4 constant internal MAGICVALUE = 0xd1f21f4f;
+    bytes4 constant internal ERC1271_MAGICVALUE = 0xd1f21f4f;
```
Comment (Brechtpd) [inline packages/loopring_v3/contracts/impl/StatelessWallet.sol:32]: It's a different magic value than ERC1271.

File: packages/loopring_v3/circuit/Utils/Data.h [hunk 2]
```diff
@@ -131,12 +131,12 @@ static auto dummySignature = R"({
 enum class TransactionType
 {
     Noop = 0,
-    SpotTrade,
     Deposit,
-    NewAccount,
     Withdrawal,
-    AccountUpdate,
     Transfer,
+    SpotTrade,
```
Comment (dong77) [inline packages/loopring_v3/circuit/Utils/Data.h:137]: TODO(daniel): also rename these const

File: packages/loopring_v3/test/types.ts [hunk 3]
```diff
@@ -235,7 +235,7 @@ export interface TxBlock {
 export interface Block {
   blockIdx: number;
   filename: string;
-  blockType: BlockType;
+  blockType: number;
```
Comment (dong77) [inline packages/loopring_v3/test/types.ts:238]: @Brechtpd we still have this 'blockType', shall we remove it?

Comment (Brechtpd) [reply packages/loopring_v3/test/types.ts:238]: Would keep it for now, could still make sense.

**Pull request merged**